spec/models/content_asset_spec.rb

```diff
@@ -1,5 +1,28 @@
 require 'rails_helper'
 
 RSpec.describe ContentAsset, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe "with an attached file" do
+    before(:each) do
+      @content_asset = FactoryBot.create(:content_asset)
```

Comment:
Given this spec is describing `ContentAsset`, the object should be initialised here instead of using `FactoryBot`.

Reply:
You could use FactoryBot attributes_for to get a hash of the attributes, then construct a ContentAsset from them.  

But Tony,  I reckon Factorybot does cause ContentAsset create to be called, and any hooks.

Reply:
This spec is to test the `content_asset` model. The object the test asserts against should be an instance of the object rather than a `FactoryBot` fixture